server/pbench/bin/pbench-server-prep-version-001

```diff
@@ -0,0 +1,182 @@
+#! /bin/bash
+
+# set -x
+###########################################################################
+# load common things
+opts=$SHELLOPTS
+case $opts in
+    *xtrace*)
+        dir=$(dirname $(which $0))
+        PROG=$(basename $(which $0))
+        ;;
+    *)
+        dir=$(dirname $0)
+        PROG=$(basename $0)
+        ;;
+esac
+
+export CONFIG=/opt/pbench-server/lib/config/pbench-server.cfg
+
+. $dir/pbench-base.sh
+###########################################################################
+
+archive=$(getconf.py archive-dir pbench-server)
+quarantine=$(getconf.py quarantine-dir pbench-server)
+duplicates=$(getconf.py duplicates-dir pbench-server)
+
+errlog=$LOGSDIR/$PROG/$PROG.error
+if [[ "$archive" == "" ]] ;then
+    echo "Failed: \"getconf.py archive-dir pbench-server" >> $errlog
+    exit 1
+fi
+
+if [[ "$quarantine" == "" ]] ;then
+    echo "Failed: \"getconf.py quarantine-dir pbench-server" $errlog
+    exit 1
+fi
+
+if [[ "$duplicates" == "" ]] ;then
+    echo "Failed: \"getconf.py duplicates-dir pbench-server" $errlog
+    exit 1
+fi
+
+log_init $(basename $0) $LOGSDIR/$(basename $0)
+
+tmp=$TMP/$PROG.$$
+
+function quarantine () {
+    dest=$1
```

Comment:
Do we need to `shift` after referencing `$1` so that `$@` doesn't include the `dest` value?

Reply:
Yes.